src/production.ts

```diff
@@ -32,11 +32,23 @@ export async function isProductionTag(): Promise<boolean> {
 
   try {
     // Get all branch names that are valid "production" branches
-    const branches = await octokit.request('GET /repos/{owner}/{repo}/branches', {
-      owner,
-      repo,
-    });
-    const branchNames = branches.data.map((branch) => branch.name).filter(isProductionBranch);
+    let page = 0;
+    const branches: any[] = [];
```

Comment:
This should be `components["schemas"]["short-branch"][]` with `components` imported from @octokit/openapi-types